### PR TITLE
#️⃣ Prefix Commands with a #

### LINKED ISSUE #277: #️⃣  Prefix commands with a hash
It may be a good idea to add a `#` character before commands like `normalize` and `print`, to avoid keyword creep as we add more.

### PR DESCRIPTION
**Pull request by TOTBWF**
## Patch Description
This PR is a small tweak that prefixes "commands" like `print` and `normalize` with a `#` to avoid keyword creep, and make them visually distinct. This closes #277.

### CODE REVIEW

**Review by jonsterling — APPROVED** [review]
Looks good! thanks!

**Comment by favonia** [comment]
My only objection is that I want to change `#t` and `#f` to something not starting with `#`.

**Comment by jonsterling** [comment]
> My only objection is that I want to change `#t` and `#f` to something not starting with `#`. I hope something is a command **if and only if** it starts with `#`.

Oohhh, that's a good point. Yes, I guess they have to be renamed. Maybe `tt/ff`.

File: src/core/Syntax.ml [hunk 1]
```diff
@@ -290,12 +290,14 @@ struct
...
-      Format.fprintf fmt "#f"
+      Format.fprintf fmt "%a"
+        Uuseg_string.pp_utf_8 "⊥"
     | Cof (Cof.Join phis) ->
       let pp_sep fmt () = Uuseg_string.pp_utf_8 fmt " ∨ " in
       Format.pp_print_list ~pp_sep (pp env P.(surrounded_by cof_join)) fmt phis
     | Cof (Cof.Meet []) ->
-      Format.fprintf fmt "#t"
+      Format.fprintf fmt "%a"
+        Uuseg_string.pp_utf_8 "⊤"
```
Comment (favonia) [inline src/core/Syntax.ml:300]: 🔝

**Review by jonsterling — CHANGES_REQUESTED** [review]
I think I don't want to use \top,\bot for true/false because this promotes the identification of booleans with propositions... It could make the cofibration language a bit confusing.

Likewise it seems like the current parser would allow `tt` to be used for the true cofibration, but I find that confusing.

My preference would be for all those to be separate if that's ok with everyone.

**Comment by TOTBWF** [comment]
Yeah, I can see how it would be somewhat misleading. I don't want to steal `top` and `bot` as keywords, as that could get kind of annoying, but I am struggling a bit to find names that are suggestive that dont:
- Steal common identifiers
- Don't suggest that propositions are booleans

**Comment by jonsterling** [comment]
Thanks for the clarification @favonia @TOTBWF! My new proposal is:

1. We have cofibrations called `true` and `false`; these can optionally be written with the top/bottom unicode symbols. It is not unexpected for these to be keywords in a language.
2. We have booleans called `tt` and `ff`; once we add inductive types, these can be deleted.

**Comment by favonia** [comment]
Revision: ✅ for true and ⛔ for false

**Comment by TOTBWF** [comment]
In order to use `true` and `false`, you must include the following command:
```
#I-acknowledge-that-propositions-are-in-fact-not-booleans
```

Perhaps something could be added to the license as well.

**Review by jonsterling — APPROVED** [review]

**Comment by favonia** [comment]
> In order to use `true` and `false`, you must include the following command:
> 
> ```
> #I-acknowledge-that-propositions-are-in-fact-not-booleans
> ```
> 
> Perhaps something could be added to the license as well.

Alternative proposal: the following binding will disable keywords `true` and `false`.
```
def _ = "use strict"
```

**Review by favonia — APPROVED** [review]
I thought I approved this PR? My memory is not working...

**Pull request merged**